rust/datafusion/Cargo.toml

```diff
@@ -61,6 +61,8 @@ futures = "0.3"
 pin-project-lite= "^0.2.0"
 tokio = { version = "0.2", features = ["macros", "blocking", "rt-core", "rt-threaded", "sync"] }
 log = "^0.4"
+md-5 = "^0.9.1"
```

Comment:
I generally agree with you, @alamb. In this case, we want to support posgres dialect, so it makes sense to support these functions (and not implement these ourselves, as they are even security related).

In general, as long as the crates are small, I do not see a major issue. Our expensive dependencies are Tokio, crossbeam, etc, specially because they really increase the compile time (e.g. compared to the arrow crate).

We already offer a scalar UDF that has the same performance as our own expressions. So, I think that this is the most we can do here.